db/models.py

```diff
@@ -50,3 +57,63 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=User, on_delete=models.CASCADE)
```

Comment:
use AUTH_USER_MODEL instead because when use change the model inside this constant, then it won't be updated here